examples/data-sources/launchdarkly_release_policy/data-source.tf

```
@@ -0,0 +1,35 @@
+# Example: Get release policy information
+data "launchdarkly_release_policy" "example" {
+  project_key = "example-project"
+  key         = "example-policy"
+}
+
+# Use the release policy information
+output "policy_details" {
+  value = {
+    name           = data.launchdarkly_release_policy.example.name
+    release_method = data.launchdarkly_release_policy.example.release_method
+    environments   = data.launchdarkly_release_policy.example.scope[0].environment_keys
+  }
+}
+
+# Example: Check if policy is guarded release
+locals {
+  is_guarded_release = data.launchdarkly_release_policy.example.release_method == "guarded-release"
+  has_rollback = length(data.launchdarkly_release_policy.example.guarded_release_config) > 0 ? 
+    data.launchdarkly_release_policy.example.guarded_release_config[0].rollback_on_regression : false
```

I think this will fix the failing test.

```suggestion
  has_rollback = length(data.launchdarkly_release_policy.example.guarded_release_config) > 0
    ? data.launchdarkly_release_policy.example.guarded_release_config[0].rollback_on_regression : false
```